src/feature/relay/routermode.h

```diff
@@ -20,4 +22,20 @@ MOCK_DECL(int, advertised_server_mode, (void));
 
 void set_server_advertised(int s);
 
+#define have_module_relay() (1)
```

Comment:
This needs a doxygen comment, since it isn't advertised elsewhere.

Reply:
Fixed in 25a509478e0a57b698fe2785ce2de3549c2702f4
Fixed a missing comment for have_module_dirauth() in e6056fa6492821b67bf749a1011bff48382d05d4